template/src/bin/main.rs

```diff
@@ -54,7 +62,7 @@ fn main() -> ! {
     //ENDIF
 
     //IF option("alloc")
-    esp_alloc::heap_allocator!(72 * 1024);
+    esp_alloc::heap_allocator!(size: 72 * 1024);
```

Comment:
we also need the changes for the async-main

Reply:
Uh! CI is not checking the async-main?? :eyes:

Reply:
We used to check it: https://github.com/esp-rs/esp-generate/actions/runs/13008839425/job/36281730799#step:7:1047

Reply:
The following options are being checked:
- empty, `alloc`, `defmt`, `esp-backtrace`, `log`, `unstable-hal`
We are missing:
- `wifi`, `ble`, `embassy`, `probe-rs`, `flashing-probe-rs/defmt`, `flashing-probe-rs/panic-rtt-target/`, `flashing-espflash/log`, `flashing-espflash/demft`, `flashing-espflash/esp-backtrace`

Reply:
Opened #130, will resolve this

Reply:
Let's check everything. CI time isn't important here, making sure we have the coverage is.